src/uwtools/resources/jsonschema/execution-serial.jsonschema

```diff
@@ -2,16 +2,7 @@
   "additionalProperties": false,
   "properties": {
     "batchargs": {
-      "additionalProperties": true,
-      "properties": {
-        "walltime": {
-          "type": "string"
-        }
-      },
-      "required": [
-        "walltime"
-      ],
-      "type": "object"
+      "$ref": "urn:uwtools:batchargs"
```

Comment:
Reference the new `batchargs.jsonschema` file, shared between this schema and `execution.jsonschema`.